### PR TITLE
add visual testing framework

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

File: .storybook-test/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const merge = require('lodash/merge');
+const path = require('path');
+const wixStorybookConfig = require('yoshi/config/webpack.config.storybook');
+
+const StylableWebpackPlugin = require('@stylable/webpack-plugin');
+const project = require('yoshi-config');
+const { resolveNamespaceFactory } = require('@stylable/node');
+
+//TODO - this should be configured inside yoshi
+function reconfigureStylable(config) {
```
Comment (shlomitc) [inline .storybook-test/webpack.config.js]: let's separate it to a common file

File: package.json [hunk 2]
```diff
@@ -25,21 +25,24 @@
     "build": "yoshi build && npm run copy-styleable-ext && npm run compile-base-ui && npm run generate-stylable-index && npm run storybook:build",
     "postbuild": "import-path --path src/components --dts",
     "pr-postbuild": "npm install @wix/teamcity-surge-autorelease@^1.0.0 --no-save && teamcity-surge-autorelease",
-    "test": "npm run test:unit && autotools-sanity && npm run test:e2e",
+    "test": "npm run test:unit && autotools-sanity && npm run test:e2e && npm run test:eyes-storybook",
     "posttest": "npm run lint",
     "test:watch": "yoshi test --jest --watch",
     "test:unit": "yoshi test --jest",
-    "test:e2e": "yoshi test --protractor",
+    "test:e2e": "concurrently 'node scripts/e2e-server.js' 'npm run test:e2e-only' --names 'e2e-server,protractor' --success first --kill-others",
```
Comment (shlomitc) [inline package.json]: why is this change needed for the visual tool?

Comment (jonathanadler) [reply package.json]: I just did a `cherry-pick` from the tabs branch, and it was part of the commit.
I think I added this for working on the e2e's locally.
Do you prefer I remove it from this PR?
